Autopilot/Path Manager/PathManager.c

```diff
@@ -667,12 +667,37 @@ void copyGPSData(){
     pmData.checkbyteDMA2 = generatePMDataDMAChecksum2();
 }
 
+// check if power on reset, brown out reset, watchdog time out or software reset flags are set
+
+ char checkForReset(){
+     if (RCONbits.POR || RCONbits.BOR || RCONbits.WDTO || RCONbits.SWR || RCONbits.EXTR){
+         // clear RCON(Reset control) bits)
+         RCONbits.BOR = 0;
```

Comment:
Point I was trying to make earlier... By resetting the flags here,  it is breaking functionality elsewhere.  Simply calling getErrorFlags() /checkErrorFlags()  would be a better way to clear/get the flags.